utility-emissions-channel/typescript_app/tests/test.js

```diff
@@ -64,76 +70,130 @@ describe("Test fabric", function() {
         orgName: "auditor1",
         utilityId: "USA_EIA_11208",
         partyId: "1234567890",
-        fromDate: "2020-04-06T10:10:09Z",
-        thruDate: "2020-04-06T10:10:09Z",
+        fromDate: "2018-05-07T10:10:09Z",
+        thruDate: "2018-05-07T10:10:09Z",
         energyUseAmount: 100,
         energyUseUom: "TONS",
       })
       .end((error, response) => {
         
-        let res = JSON.stringify(response.body.info.includes("Failed to submit transaction"));
-        if(res)
-        {
-        assert(response.status == 409, "Could not reach the gateway network");
+       try{
+        //console.log(`response :  ${JSON.stringify(response.body)}`);
+        expect(response.body.info).to.equal("EMISSION RECORDED TO LEDGER");
         done();
-        }
+       }
         
-        else if(response.body.info == "EMISSION RECORDED TO LEDGER")
+       catch (error)
         {
-        assert(response.body.info == "EMISSION RECORDED TO LEDGER", "Failed to record emission.");
-        done();
-        }
-        
-        else if(error) 
+          done(error);
+        } 
+       
+      });
+  });
+ 
+ 
+  it("should display emission records for the correct year (2019)", function(done) {
+    chai
+       .request(host)
+      .post(recordEmissionPath)
+      // .field('myparam' , 'test')
+      .set("content-type", "application/x-www-form-urlencoded")
+      .send({
+        userId: "TestUser",
+        orgName: "auditor1",
+        utilityId: "USA_EIA_11208",
+        partyId: "1234567890",
+        fromDate: "2019-04-06T10:10:09Z",
+        thruDate: "2019-04-06T10:10:09Z",
+        energyUseAmount: 100,
```

Comment:
Hi @sichen1234 ,

Did you mean this ?
I am first recording the emission for year 2019, and then checking if the response has the emission recorded for year 2019.